File: backend/src/main/java/com/eskisehir/eventapi/domain/model/SensitivityLevel.java
package com.eskisehir.eventapi.domain.model;

/**
 * Reusable sensitivity/tolerance level.
 * Used for budget sensitivity and crowd tolerance in user preferences.
 */
public enum SensitivityLevel {
    LOW("Düşük"),
    MEDIUM("Orta"),
    HIGH("Yüksek");

    private final String displayNameTr;

    SensitivityLevel(String displayNameTr) {
        this.displayNameTr = displayNameTr;
    }

    public String getDisplayNameTr() {
        return displayNameTr;
    }
}

File: mobile/app/src/main/java/com/eskisehir/eventapp/data/model/SensitivityLevel.kt
package com.eskisehir.eventapp.data.model

/**
 * Reusable sensitivity/tolerance level.
 * Used for budget sensitivity and crowd tolerance in user preferences.
 */
enum class SensitivityLevel(val displayNameTr: String) {
    LOW("Düşük"),
    MEDIUM("Orta"),
    HIGH("Yüksek")
}
